soccer/gameplay/skills/capture.py

```diff
@@ -1,270 +1,302 @@
-import single_robot_behavior
+import single_robot_composite_behavior
 import behavior
 from enum import Enum
 import main
-import evaluation
-import constants
 import role_assignment
 import robocup
-import planning_priority
-import time
-import skills.move
-
+import constants
+import skills.settle
+import skills.collect
+import math
+import evaluation.ball
 
-class Capture(single_robot_behavior.SingleRobotBehavior):
-    # Speed in m/s at which a capture will be handled by coarse and fine approach instead of intercept
-    InterceptVelocityThresh = 0.15
 
-    # Multiplied by the speed of the ball to find a "dampened" point to move to during an intercept
-    DampenMult = 0.0
+class Capture(single_robot_composite_behavior.SingleRobotCompositeBehavior):
 
-    # The distance to transition from coarse approach to fine
-    # TODO: The correct way to do this would be using our official max acceleration and current speed
-    CoarseToFineApproachDistance = 0.3
+    DRIBBLE_SPEED = 50
```

Comment:
I want these to be that `constants.StandardPower`, but at the current moment, if you bump up the dribbler, the robot ends up sucking up the ball and lifting it's own wheels off the ground. At the current moment, I'm leaving them separate so we can bump it up for the initial settle, the lower it for control portions